vnext/Microsoft.ReactNative/getLayoutProps.ps1

```diff
@@ -1,5 +1,5 @@
 # This script produces the values for the layoutProperties vector
 # in TestHook.cpp
-$page = Invoke-WebRequest "https://reactnative.dev/docs/layout-props"
+$page = Invoke-WebRequest -UseBasicParsing "https://reactnative.dev/docs/layout-props"
```

Comment:
This doesn't hurt, but this looks like a one-off script to produce these. I'm not actually sure we should have this file at all.

Reply:
I"ll leave as-is.